packages/components/package.json

```diff
@@ -41,8 +41,9 @@
     "react": ">= 16.8.0"
   },
   "dependencies": {
-    "@chanzuckerberg/czedi-kit-styles": "^0.0.1-alpha.5",
     "@chanzuckerberg/czedi-kit-tokens": "^0.0.1-alpha.5",
+    "autoprefixer": "^10.1.0",
+    "postcss": "^8.2.1",
```

Comment:
peer dependencies

Reply:
Is this because postcss is part of twin-macro, or styled-components? I'm mainly concerned about this becoming part of the included dependencies, makes more sense as a dev dependency if anything

Reply:
they're both from `tailwindcss` -- you're totally right they should be dev dependencies though